shared/lending-protocols.ts

```diff
@@ -0,0 +1,15 @@
+export enum LendingProtocol {
+    AaveV2 = 'aavev2',
+    AaveV3 = 'aavev3',
+    Ajna = 'ajna',
+    Maker = 'maker',
+    SparkV3 = 'sparkv3',
+}
+
+export enum LendingProtocolLabel {
+    aavev2 = 'Aave V2',
+    aavev3 = 'Aave V3',
+    ajna = 'Ajna',
+    maker = 'Maker',
+    sparkv3 = 'Spark',
```

Comment:
label could be an object, doesn`t make sense to be an enum as it`s mapping from LendingProtocol enum
```ts
export const LendingProtocolLabel = {
    [LendingProtocol.AaveV3] = 'Aave V2',
   ...etc
}
```